package.json

```diff
@@ -142,6 +142,7 @@
     "vscode-builtin-xml": "https://github.com/theia-ide/vscode-builtin-extensions/releases/download/v1.39.1-prel/xml-1.39.1-prel.vsix",
     "vscode-builtin-yaml": "https://github.com/theia-ide/vscode-builtin-extensions/releases/download/v1.39.1-prel/yaml-1.39.1-prel.vsix",
     "vscode-editorconfig": "https://github.com/theia-ide/editorconfig-vscode/releases/download/v0.14.4/EditorConfig-0.14.4.vsix",
-    "vscode-eslint": "https://github.com/theia-ide/vscode-eslint/releases/download/release%2F2.0.15/vscode-eslint-2.0.15.vsix"
+    "vscode-eslint": "https://github.com/theia-ide/vscode-eslint/releases/download/release%2F2.0.15/vscode-eslint-2.0.15.vsix",
+    "vscode-references-view": "https://download.jboss.org/jbosstools/vscode/3rdparty/vscode-references-view/references-view-0.0.47-05339d.vsix"
```

Comment:
@vrubezhny is there a reason for not using the official [release](https://github.com/microsoft/vscode-references-view/releases/download/0.0.4/references-view-0.0.4.vsix) from the GitHub [repository](https://github.com/microsoft/vscode-references-view)?

Reply:
> @vrubezhny is there a reason for not using the official [release](https://github.com/microsoft/vscode-references-view/releases/download/0.0.4/references-view-0.0.4.vsix) from the GitHub [repository](https://github.com/microsoft/vscode-references-view)?

The latest release there is v.0.0.4:  https://github.com/microsoft/vscode-references-view/releases - which is not suitable for us. 
Latest tag is v.0.0.30, which is still far away of current v0.0.47 which we plan to include. Also there is no VSIX archives attached to a `tagged` version of the plugin on GitHub: https://github.com/microsoft/vscode-references-view/releases/tag/v0.0.30

Reply:
Should we do like other extensions (ex: [vscode-eslint](https://github.com/theia-ide/vscode-eslint/releases)) and fork the repository and do the releases ourselves? I assume you ran `vsce package` and you uploaded the `.vsix` to the jboss downloads page?

Reply:
yes, we should have one way

Reply:
@vince-fugnitto How do you do release that includes vsix file?

Reply:
> @vince-fugnitto How do you do release that includes vsix file? Do you upload it?

I build the release myself using `vsce package`, and perform the release myself using the same tag name as the version I checked out.

Reply:
@vrubezhny Did you build vsix from this commit https://github.com/theia-ide/vscode-references-view/commit/05339da0c4d62d9804f9e33e47cafc2fcf1b2afa?